README.md

```diff
@@ -160,3 +161,5 @@ It may be possible to reduce build times by making use of caches that persist on
 ### Can I run multiple agents on a single host?
 
 Yes, by running multiple replicas of the launch-agent with unique names, it is possible to run as many agents (and therefore jobs) on a single host as you desire. However, care must be taken to ensure that these jobs are sufficiently isolated from each other that they don't conflict if run at the same time.
+
+[circleci/runner]: https://hub.docker.com/repository/docker/circleci/runner
```

Comment:
This link requires authentication. However, `https://hub.docker.com/r/circleci/runner` seems to work fine. Should we use that one instead?

Reply:
well spotted, thanks @RicardoLinck